Index: sipwitch-1.2.3/configure.ac
===================================================================
--- sipwitch-1.2.3.orig/configure.ac
+++ sipwitch-1.2.3/configure.ac
@@ -153,7 +153,7 @@ AC_ARG_WITH(netscripting,
 # and cgi bin path
 AC_ARG_WITH(cgibindir,
     AC_HELP_STRING([--with-cgibindir=path], [specific cgi-bin path]), [
-        cgibindir="$with_initrddir"
+        cgibindir="$with_cgibindir"
 ])
 
 if test -z "$cgibindir" ; then
